File: typist.py
def typist(s):
    capsLock = 0
    count = len(s)
    if s[0] == s[0].upper():
        capsLock += 1
    for ele in range(0, (len(s)-1)):
        if s[ele] == s[ele].lower() and s[ele+1] == s[ele+1].upper():
            capsLock += 1
        elif s[ele] == s[ele].upper() and s[ele+1] == s[ele+1].lower():
            capsLock += 1
            
    return count + capsLock

File: typist.js
function typist(s){
    let capsLock = 0;
     let count= s.length;
     if (s[0] === s[0].toUpperCase()){
       capsLock++;
     }
     for (let i = 0; i < s.length-1; i++){
       let letter = s[i];
       if ( ( (s[i] == s[i].toLowerCase()) && (s[i + 1] == s[i + 1].toUpperCase()) ) || 
           ( (s[i] == s[i].toUpperCase()) && (s[i + 1] == s[i + 1].toLowerCase())) ){
         capsLock++;
       }
     }
     return count + capsLock;
   }